test/dynamo/test_higher_order_ops.py

```diff
@@ -2370,7 +2372,68 @@ def fn(pred, pytree_in):
                 torch.compile(fn, backend="eager")(pred, pytree_in)
 
 
+class HigherOrderOpLoggingTests(LoggingTestCase):
```

Comment:
Nit: are these really Logging tests?

Reply:
No! I've changed the class name to `HigherOrderOpVmapGuardTests`